sub-packages/bionemo-evo2/src/bionemo/evo2/run/predict.py

```diff
@@ -130,6 +131,13 @@ def parse_args():
         "know a model was trained with a specific interpolation factor for ROPE, provide it here, it can make a big "
         "difference in accuracy.",
     )
+    ap.add_argument(
+        "--lora-checkpoint-path",
```

Comment:
Consider expanding the help text to give users a clearer idea of what happens when they pass a LoRA checkpoint path, that it inits model transform. For example, you could briefly mention what is restored and how it affects the model
Also, for consistency and readability, you might want to use the common capitalization LoRA in the description